### ISSUE TITLE
程序员学深度学习快速入门五步法

### ISSUE THREAD

**Issue by qiwihui** [qiwihui/pocket_readings#504]
&#31243;&#24207;&#21592;&#23398;&#28145;&#24230;&#23398;&#20064;&#24555;&#36895;&#20837;&#38376;&#20116;&#27493;&#27861;<br>
4&#31181;&#22522;&#26412;&#20803;&#32032;&#65306; 1. &#32593;&#32476;&#32467;&#26500;&#65306;&#30001;10&#31181;&#22522;&#26412;&#23618;&#32467;&#26500;&#21644;&#20854;&#20182;&#23618;&#32467;&#26500;&#32452;&#25104; 2. &#28608;&#27963;&#20989;&#25968;&#65306;&#22914;relu, softmax&#12290;&#21475;&#35776;: &#26368;&#21518;&#36755;&#20986;&#29992;softmax&#65292;&#20854;&#20313;&#22522;&#26412;&#37117;&#29992;relu 3.<br>
<br>
Tags: tensorflow<br>
<br>
via Pocket https://ift.tt/2JyKsTp<br>
<br>
June 13, 2018 at 05:11PM
